key_action_map.cpp

```diff
@@ -455,8 +455,9 @@ void test_key_action_map()//!OCLINT tests can be many
     assert(kam.to_key(action_type::turn_right) == sf::Keyboard::D);
     assert(kam.to_key(action_type::accelerate_forward) == sf::Keyboard::W);
     assert(kam.to_key(action_type::accelerate_backward) == sf::Keyboard::S);
-    assert(kam.to_key(action_type::shoot) == sf::Keyboard::Q);
-    assert(kam.to_key(action_type::shoot_stun_rocket) == sf::Keyboard::E);
+    // Note that this line is commented out due to the need of minimal viable game, ref to (644)
+    //assert(kam.to_key(action_type::none) == sf::Keyboard::Q);
```

Comment:
I don't understand why you turn the link to Q off, but not E?
Also, commenting out code is a sloppy policy, version control has your back so be bold and delete entirely if you don't need it 😎

Reply:
Will delete those lines asap, but I will be unavailable until 17th.

I think I turned off both Q and E, if I understand the minimal viable game goal correctly

Reply:
Yes, no rush :)
I can also do it on your behalf, let me suggest a change.
But I'm still unsure what you want to do exactly. If it's just deactivating the rockets, you can keep keys Q and E, mapped to `action_type::none`. That should mean that pressing those keys triggers... nothing! right?

Reply:
Yes, and I think this will be the neatest way to disable these two keys, without breaking to many things